Packs/CortexXpanse/Integrations/CortexXpanse/CortexXpanse.py

```diff
@@ -0,0 +1,658 @@
+from typing import Any, Dict, List, cast, Tuple
+
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+DEFAULT_SEARCH_LIMIT = 100
+MAX_ALERTS = 100  # max alerts per fetch
+TIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
+
+
+class Client(BaseClient):
+    """
+    Client class to interact with the service API.
+    """
+
+    def __init__(self, base_url, verify, proxy, headers, auth):
+        """
+        Class initialization.
+        """
+        super().__init__(base_url, verify=verify, proxy=proxy, headers=headers, auth=auth)
+
+    def list_alerts_request(self, request_data: Dict) -> Dict[str, Any]:
+        """Get a list of all asm alerts '/alerts/get_alerts/' endpoint.
+
+        Args:
+            request_data (dict): dict of parameters for API call.
+
+        Returns:
+            dict: dict containing list of external services.
+        """
+        headers = self._headers
+
+        response = self._http_request('POST', '/alerts/get_alerts/',
+                                      json_data=request_data, headers=headers)
+
+        return response
+
+    def list_external_service_request(self, search_params: List[Dict]) -> Dict[str, Any]:
+        """Get a list of all your external services using the '/assets/get_external_services/' endpoint.
+
+        Args:
+            search_params (list): list of search parameters to add to the API call body.
+
+        Returns:
+            dict: dict containing list of external services.
+        """
+        data = {"request_data": {"filters": search_params, "search_to": DEFAULT_SEARCH_LIMIT}}
```

Comment:
@adi88d , I mentioned that this integration is a copy of https://github.com/demisto/content/tree/master/Packs/CortexAttackSurfaceManagement/Integrations/CortexAttackSurfaceManagement and adding fetch and /get_alerts.  Because that behavior is in that integration, i prefer to keep it.

Reply:
ok